src/components/bark/login/layout.tsx

```diff
@@ -0,0 +1,70 @@
+import Image from "next/image";
+import { BarkH1 } from "../bark-typography";
+import { BarkNavRoute } from "../bark-nav";
+import Link from "next/link";
+
+const LoginLayout = ({
+  accountType,
+  children,
+}: Readonly<{ accountType: string; children: React.ReactNode }>) => {
+  let logoSrc = "";
+
+  if (accountType === "ADMIN") {
+    logoSrc = "/purpleDogHouse.svg";
+  } else if (accountType === "VET") {
+    logoSrc = "/pawPrint.svg";
+  } else if (accountType === "USER") {
+    logoSrc = "/orangeDogHouse.svg";
+  }
+
+  const footerRoutes: BarkNavRoute[] = [
+    {
+      label: "Contact us",
+      href: "/contact-us",
+    },
+    {
+      label: "Privacy Policy",
+      href: "/privacy-policy",
+    },
+    {
+      label: "Terms & Conditions",
+      href: "/terms-and-conditions",
+    },
+  ];
+
+  return (
+    <div className="relative mt-40 flex min-h-screen w-full flex-col justify-between">
```

Comment:
Outer layout of the Navbar is still not incorporated in the design.

Delegate to main layout component since it is found in all pages.

Reply:
What does this mean? 😅

Reply:
![Screenshot 2024-03-03 at 10 11 03](https://github.com/barkbank/barkbank-pawtal/assets/48792128/bbbeed0b-a258-4ea3-a502-521268ea7a27)

I realised that was quite vague 😅

I was referring to the current implementation not having this Navbar incorporated yet.

Reply:
oic. Thanks for the clarification.

I created a [ticket for nav and footer](https://www.notion.so/Add-BarkNav-and-BarkFooter-to-root-layout-000e71d6a78a4a5bb64191eacf6a0201?pvs=4) this morning. We can defer nav and footer to that ticket.